### PR TITLE
Changes for View Mileage page and Audit Trail

### PR DESCRIPTION
**Pull request by mvaishnavi**

### CODE REVIEW

**Comment by mvaishnavi** [comment]
My Reports - View Mileage
Screenshot 2022-07-25 at 5 39 59 PM

Audit trail
Screenshot 2022-07-25 at 5 40 31 PM

Team Reports - View Mileage
Screenshot 2022-07-25 at 5 41 25 PM

**Review by Dimple16 — APPROVED** [review]

File: src/app/shared/pipes/mileage-rate-name.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'mileageRateName',
+})
+export class MileageRateName implements PipeTransform {
+  transform(value) {
+    if (!value) {
+      return value;
```
Comment (Julias0) [inline src/app/shared/pipes/mileage-rate-name.pipe.ts:9]: return '';

File: src/app/shared/pipes/mileage-rate-name.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'mileageRateName',
+})
+export class MileageRateName implements PipeTransform {
+  transform(value) {
```
Comment (Julias0) [inline src/app/shared/pipes/mileage-rate-name.pipe.ts:7]: transform(value: string): string {

File: src/app/shared/pipes/mileage-rate-name.pipe.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      two_wheeler: 'Two Wheeler',
+      four_wheeler: 'Four Wheeler - Type 1',
+      four_wheeler1: 'Four Wheeler - Type 2',
+      four_wheeler3: 'Four Wheeler - Type 3',
+      four_wheeler4: 'Four Wheeler - Type 4',
+      bicycle: 'Bicycle',
+      electric_car: 'Electric Car',
+    };
+
+    return names[value] ? names[value] : value;
```
Comment (Julias0) [inline src/app/shared/pipes/mileage-rate-name.pipe.ts:22]: `return names[value] || value;`

**Review by Julias0 — APPROVED** [review]
Minor comments - fix and merge

**Comment by mvaishnavi** [comment]
will merge to master along with - https://github.com/fylein/fyle-mobile-app/pull/1268

**Pull request closed without merging**